socorro/unittest/external/boto/__init__.py

```diff
@@ -26,7 +29,7 @@ def get_config(cls, values_source=None):
         app_name="testapp",
         app_version="1.0",
         app_description="",
-        values_source_list=[values_source],
+        values_source_list=[environment, values_source],
```

Comment:
This changes tests such that they use environment variables from `docker/config/test.env` rather than who knows what.